docs/Concepts/Node-Keys.md

```diff
@@ -1,34 +1,42 @@
 ---
-description: Besu private and public key used to identify node
+description: Private and public key, and node address used to identify nodes
 ---
 
-# Node keys
+# Node keys and node address
 
-Each node has a node key pair consisting of a node private key and a node public key.
+Each node has a private / public key pair and a node address. Besu uses the private / public key
+pair to sign and verify transactions, and the node address as an identifier for the node.
 
 ## Node private key
 
-If a `key` file does not exist in the data directory and when starting Besu the
+When starting Hyperledger Besu, if the
 [`--node-private-key-file`](../Reference/CLI/CLI-Syntax.md#node-private-key-file) option is not
-specified, Besu generates a node private key and writes it to the `key` file.
-Stopping and restarting Besu without deleting the `key` file causes Besu to reuse the same private
-key when restarting.
+specified and a `key` file does not exist in the data directory for the node, Besu generates a node
+private key and writes it to the `key` file.
 
-If a `key` file exists in the data directory when starting Besu, the node starts with the private
-key in the `key` file.
+If a `key` file does exist in the data directory when starting Besu, the node starts with using
+private key in the `key` file.
 
 !!!info
 
     The private key is not encrypted.
 
 ## Node public key
 
-The node public key displays in the log after starting Besu. Use the
-[`public-key`](../Reference/CLI/CLI-Subcommands.md#public-key) subcommand to export the public key
-to a file.
+The node public key (`key.pub` in the data directory for the node) displays in the log after
+starting Besu. Also refered to as the node ID, Besu uses the node public key to generate the node
```

Comment:
I think the _Besu uses the node public key to generate the node address_ part of the sentence isn't really needed here because you've added in the following Node Address section.  I had to read the sentence a couple of times as it's written currently.  

Suggest simplifying it to:
Also referred to as the node ID, the node public key forms part of the enode URL for a node.

Reply:
Done.